bin/eslint.js

```diff
@@ -4,4 +4,8 @@ const CODE_DIR = "/code";
 process.chdir(CODE_DIR);
 
 const ESLint = require("../lib/eslint");
-ESLint.run({ dir: CODE_DIR });
+const issues = ESLint.run({ dir: CODE_DIR });
```

Comment:
Talked this through with @GordonDiggs IRL, and I'm afraid that would be a showstopper in this case. At points in the past we did do more buffering of issues in this engine, and large repos with many issues could bring the engine to its knees.

As I was writing this, I see it looks like you already addressed this with #230, though, so that's great. Thanks!